features/users/user_change_email.feature

```diff
@@ -0,0 +1,77 @@
+@users
+Feature:
+  As a registered user
+  I should be able to change my email
```

Comment:
The enthusiasm regarding tests is appreciated, however all these things except for the "Translated email is sent when email is changed" scenario are already tested in`features/other_a/profile_edit.feature` :/

I would suggest to move the translated email scenario to that file and remove the rest of the new tests.